### ISSUE TITLE
Not working with PS 1.6.0.11

### ISSUE THREAD

**Issue by canggut** [Lebioda/PrestaShop-modules-CleanURLs#11]
Module installed,
Advanced Parameters > Performance > Clear Cache,
Preferences -> SEO and URL -> Set userfriendly URL off -> Save,
Preferences -> SEO and URLs -> Set userfriendly URL on -> Save,

But still: Keyword "{id}" required

**Comment by gyan111** [comment]
Yes the same issue. I think the module has not been updated for a long time and there may have some functionality changed on prestashop.
